app/src/main/java/me/hufman/androidautoidrive/carapp/carinfo/CarDetailedInfo.kt

```diff
@@ -164,6 +173,14 @@ class CarDetailedInfo(carCapabilities: Map<String, Any?>, cdsMetrics: CDSMetrics
 	val rawHeading = cdsMetrics.rawHeading.format("%.0f Raw Heading")
 
 	// categories
+	private val sportFields: List<Flow<String>> = listOf(
+			engineTemp, oilTemp,
+			accelContact, brakeState,
+			engineRpm, torque,
+			drivingGearLabel , steeringAngle,
+			tempInterior, tempExterior,
```

Comment:
This page is a little too long and scrolls, but the car doesn't show a good indication that it has more data after the bottom. I think the tempInterior and tempExterior rows shouldn't be shown on the Sport Display screen, since they are already on the main overview and will be on the dedicated HVAC page in #710 , and then all of rows will fit on a single page.